fe/fe-core/src/main/java/com/starrocks/analysis/GrantRoleStmt.java

```diff
@@ -0,0 +1,73 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+
+import com.starrocks.catalog.Catalog;
+import com.starrocks.cluster.ClusterNamespace;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.common.FeNameFormat;
+import com.starrocks.common.UserException;
+import com.starrocks.mysql.privilege.Auth;
+import com.starrocks.mysql.privilege.PrivPredicate;
+import com.starrocks.qe.ConnectContext;
+
+// GRANT Role 'role' TO 'user'
+public class GrantRoleStmt extends DdlStmt {
```

Comment:
The new Analyzer framework should be used. And the new AST file should be located under the sql/ast directory. In the future, starrocks/analysis/xx will be gradually replaced by /sql/ast